hybrisprovider.cpp

```diff
@@ -520,6 +520,10 @@ void HybrisProvider::setLocation(const Location &location)
     }
 
     m_currentLocation = location;
+    Accuracy acc;
+    acc.setHorizontal(location.accuracy().horizontal());
+    acc.setVertical(location.accuracy().vertical());
+    m_currentLocation.setAccuracy(acc);
```

Comment:
This looks to be an issue within the copy constructor of the LocationData (locationtypes.h). 
Adding following to the 58 of the locationtypes fixes copy constructor.

```
direction(other.direction), climb(other.climb), accuracy(other.accuracy)
```
So, accuracy after climb.

Reply:
I will test that for you if you need me to

Reply:
Awesome, thank you

Reply:
@rainemak It seems that's it's working! To remove my previous commit you can squash changes while merging :)

Reply:
Thank you @Michal-Szczepaniak !